### ISSUE TITLE
[ CE 2.0.7 ] Add grouped product to the cart

### ISSUE THREAD

**Issue by mhabou** [magento/magento2#5089]
## Steps to reproduce
1. Install Magento 2.0.7
2. Add Grouped Product to the cart.
   ![snapshot6](https://cloud.githubusercontent.com/assets/13301885/16153141/8fc81320-3494-11e6-938b-bf799f949579.png)
3. Click on the cart icon to proceed the purchase
   ![snapshot7](https://cloud.githubusercontent.com/assets/13301885/16153169/b53b7048-3494-11e6-9507-c50037759ce5.png)
## Expected result
1. Go to checkout page
2. ...
## Actual result
1.  no response ..

**Comment by irenelagno** [comment]
@Dayssam Thank you for reporting this issue. Currently, we were unable to reproduce it with the provided information. Could you please provide more detailed steps to reproduce, e. g. settings you have changed or extra details about simple and grouped product configuration, information from the errors in browser console(if it exists)?

**Comment by mhabou** [comment]
I dont get this problem with all grouped products

On the load of the page i got [just once]

```
Error: Script error for: Magento_Checkout/js/sidebar
http://requirejs.org/docs/errors.html#scripterror
makeError()
 require.js:166
newContext/context.onScriptError()
 require.js:1681
 require.js:166:17
Error: Script error for: Magento_Ui/js/form/form
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: Magento_Customer/js/action/login
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: Magento_Customer/js/model/authentication-popup
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: mage/url
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: Magento_Ui/js/modal/alert
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: Magento_Captcha/js/view/checkout/defaultCaptcha
http://requirejs.org/docs/errors.html#scripterror
 require.js:166:17
Error: Script error for: Magento_Captcha/js/model/captchaList
http://requirejs.org/docs/errors.html#scripterror
```

When i start changing the product quantities i got 

```
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead. set-of-sprite-yoga-straps.html
TypeError: $(...).attr(...) is undefined
._getElementData()
 add-to-wishlist.js:94
$.widget/</proxiedPrototype[prop]</<()
 jquery-ui.js:402
._updateWishlistData/<()
 add-to-wishlist.js:53
.each()
 jquery.js:384
jQuery.prototype.each()
 jquery.js:137
._updateWishlistData()
 add-to-wishlist.js:43
$.widget/</proxiedPrototype[prop]</<()
 jquery-ui.js:402
handlerProxy()
 jquery-ui.js:702
jQuery.event.dispatch()
 jquery.js:4623
jQuery.event.add/elemData.handle()
 jquery.js:4292
```

Then i got no response when i click on " Go to checkout " and no products details (i got that error just in the case of the sample data grouped product " Set of Sprite Yoga Straps " )

![snapshot7](https://cloud.githubusercontent.com/assets/13301885/16153169/b53b7048-3494-11e6-9507-c50037759ce5.png)

**Comment by mhabou** [comment]
@irenelagno : 
When i added 2 groped products ( 1 from sample data & the other which i created)

when i go to checkout page i got a blank page with just the right side bar

![snapshot12](https://cloud.githubusercontent.com/assets/13301885/16237591/a797cd20-37cc-11e6-8ad2-885e91da6680.png)

```
TypeError: value is undefined
.max_text_length<()
 rules.js:56
validate()
 validator.js:18
validator/<()
 validator.js:40
_.any()
 underscore.js:250
validator()
 validator.js:39
.validate()
 abstract.js:245
.onUpdate()
 abstract.js:267
ko_subscribable_fn.notifySubscribers()
 knockout.js:1103
ko.observable/observable.valueHasMutated()
 knockout.js:1300
observable()
 knockout.js:1285
setNested()
 objects.js:43
.nested()
 objects.js:117
Element<.set()
 element.js:294
updateValue()
 links.js:57
trigger/<()
 events.js:85
forEach()
 self-hosted:216
trigger()
 events.js:82
.trigger()
 events.js:159
Element<._notifyChanges/<()
 element.js:413
forEach()
 self-hosted:216
Element<._notifyChanges()
 element.js:412
Element<.set()
 element.js:297
.initialize/<()
 shipping.js:104
Events.prototype.wait()
 events.js:60
Registry.prototype.get()
 registry.js:67
async()
 registry.js:34
.initialize()
 shipping.js:101
.wrapSuper/<()
 wrapper.js:109
createConstructor/constr()
 class.js:48
initComponent()
 layout.js:64
jQuery.Callbacks/fire()
 jquery.js:3099
jQuery.Callbacks/self.fireWith()
 jquery.js:3211
.Deferred/</deferred[tuple[0]]()
 jquery.js:3301
jQuery.Callbacks/fire()
 jquery.js:3099
jQuery.Callbacks/self.fireWith()
 jquery.js:3211
.Deferred/</deferred[tuple[0]]()
 jquery.js:3301
loadSource/<()
 layout.js:57
newContext/context.execCb()
 require.js:1650
newContext/Module.prototype.check()
 require.js:866
newContext/Module.prototype.enable/</<()
 require.js:1113
bind/<()
 require.js:132
newContext/Module.prototype.emit/<()
 require.js:1156
each()
 require.js:57
newContext/Module.prototype.emit()
 require.js:1155
newContext/Module.prototype.check()
 require.js:917
newContext/Module.prototype.enable/</<()
 require.js:1113
bind/<()
 require.js:132
newContext/Module.prototype.emit/<()
 require.js:1156
each()
 require.js:57
newContext/Module.prototype.emit()
 require.js:1155
newContext/Module.prototype.check()
 require.js:917
newContext/Module.prototype.enable/</<()
 require.js:1113
bind/<()
 require.js:132
newContext/Module.prototype.emit/<()
 require.js:1156
each()
 require.js:57
newContext/Module.prototype.emit()
 require.js:1155
newContext/Module.prototype.check()
 require.js:917
newContext/Module.prototype.enable/</<()
 require.js:1113
bind/<()
 require.js:132
newContext/Module.prototype.emit/<()
 require.js:1156
each()
 require.js:57
newContext/Module.prototype.emit()
 require.js:1155
newContext/Module.prototype.check()
 require.js:917
newContext/Module.prototype.enable()
 require.js:1143
newContext/Module.prototype.init()
 require.js:774
callGetModule()
 require.js:1170
newContext/context.completeLoad()
 require.js:1544
newContext/context.onScriptLoad()
 require.js:1671
```

**Comment by irenelagno** [comment]
@Dayssam, thank you for extra information regarding configuration. The bug about grouped product was reported in the https://github.com/magento/magento2/issues/3728 . Fix for this issue has been delivered to the develop branch with the commit https://github.com/magento/magento2/commit/520f76d054a3fd1bda57144e17e58e116361a6d5.  The second issue was reported in https://github.com/magento/magento2/issues/4921. We have created an internal ticket MAGETWO-47240 to fix the issue. Thank you.

**Comment by mcspronko** [comment]
Hi @irenelagno 

Do you have any date for MAGETWO-50368 issue?

Thanks

**Comment by king-themes** [comment]
The same with me on v2.1.0. Any way to fix it?

Thanks

**Comment by magento-engcom-team** [comment]
Hi @Dayssam
Fix for the issue seems to be released to `2.2` and `2.1` with the commit 520f76d054a3fd1bda57144e17e58e116361a6d5
Closing this report for now. Please reopen if the issue persists.
Thank you
